slim_test.go

```diff
@@ -350,3 +350,20 @@ func TestNoRaw(t *testing.T) {
 		t.Fatalf("expected %v but %v", expect, got)
 	}
 }
+
+func TestAttr(t *testing.T) {
+    tmpl, err := ParseFile("testdir/test_attr.slim")
+    if err != nil {
+        t.Fatal(err)
+    }
+    var buf bytes.Buffer
+    err = tmpl.Execute(&buf, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Thank your contribution. Please do go fmt.

Reply:
Sorry, Im do it.